config.js

```diff
@@ -8,6 +8,7 @@ System.config({
 
   map: {
     "aurelia-polyfills": "npm:aurelia-polyfills@1.1.0",
+    "interact": "npm:interact.js@1.2.6",
     "interact.js": "npm:interact.js@1.2.6",
```

Comment:
we can probably remove this line, will look at that next week as it isn't blocking/ causing issues. Will move it from typings to 2.0 @types.